### PR TITLE
updated the installlation guide

### PR DESCRIPTION
**Pull request by AkshitRawat1207**
### Description
Versions of Node and npm are updated.

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -46,22 +46,15 @@ Before you start following the guidelines, make sure to go through the [prerequi
    ```
     If you are still getting the error --hide-module then revert all changes by `git checkout .`
 
-	**Note:** Check the version of laravel-mix in package-lock.json it should be 5.0.9
-    ```sh
-     npm install
-     npm run dev
-    ```
-    **Note:** if laravel-mix version in package-lock.json changes to version 6 then npm run dev will throw error, so it should be at version 5 only.
-
```
Comment (Ayush8923) [inline docs/installation.md:55]: Why did you remove these lines? 
Any reason?

**Comment by Ayush8923** [comment]
@AkshitRawat1207 Please put the label `ready-for-review` then assign the PR for review.

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
Added one comments.

File: docs/installation.md (hunk 1 above)
Comment (GauravGusain98) [reply docs/installation.md:55]: Because we have updated the node version so this is not required

**Review by GauravGusain98 — APPROVED** [review]

**Pull request merged**